src/main/java/org/jvnet/hudson/tools/versionnumber/VersionNumberBuilder.java

```diff
@@ -316,6 +333,27 @@ private VersionNumberBuildInfo incBuild(AbstractBuild build, Map<String, String>
             }
             buildsToday = ((newVal >= 0) ? newVal : buildsToday);
         }
+        if (!this.oBuildsThisWeek.equals("")) {
```

Comment:
Please change this line to the following:
`        if (this.oBuildsThisWeek == null || !this.oBuildsThisWeek.equals("")) {`